py_mmd_tools/nc_to_mmd.py

```diff
@@ -586,6 +586,24 @@ def get_metadata_identifier(self, mmd_element, ncin, require_uuid=True):
                     '%s ACDD attribute is missing - created metadata_identifier MMD element as uuid.' %acdd)
         return id
 
+    def get_related_dataset(self, mmd_element, ncin):
+        acdd_ext_relation_type = mmd_element['relation_type']['acdd_ext']
+        relation_types = []
+        if acdd_ext_relation_type in ncin.ncattrs():
+            relation_types = self.separate_repeated(True, eval('ncin.%s' %acdd_ext_relation_type))
+        acdd_ext_id = mmd_element['id']['acdd_ext']
+        ids = []
+        if acdd_ext_id in ncin.ncattrs():
+            ids = self.separate_repeated(True, eval('ncin.%s' %acdd_ext_id))
+        assert len(relation_types) == len(ids)
```

Comment:
Do we want asset here?

Hm, we want the library to fail if this condition is not meet

Reply:
I think so - it becomes difficult if those have different lengths...